src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/formViewNetworkWideValidationSchema.tsx

```diff
@@ -11,7 +11,11 @@ import {
   isNotCatchAllIPAddress,
   getIpIsNotNetworkOrBroadcastAddressSchema,
 } from '../../commonValidationSchemas';
+
 const REQUIRED_MESSAGE = 'A value is required';
+const MUST_BE_A_NUMBER = 'Must be a number';
+const NUMBER_REGEX = /^[0-9]*$/;
+const NUMBER_EXPONENTIAL = /[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)/;
```

Comment:
Isn't `NUMBER_EXPONENTIAL` regular expression unnecessary? If we ensure from the first RegExp that the value only contains digits, it cannot be an exponential as it would use the `e` character.

So we can have `const ONLY_DIGITS_REGEX = /^\d+$/;` (with the plus it must contain at least one, and it also must end in digits),

Reply:
True, changed it